app/labor/hex_comparer.rb

```diff
@@ -1,49 +1,49 @@
 class HexComparer
+  RGB_REGEX = /^#?(?<r>..)(?<g>..)(?<b>..)$/.freeze
```

Comment:
This is now a regex with named captured groups, so we don't have to manually assemble rgb hashes.